app/controllers/items_controller.rb

```diff
@@ -36,7 +36,7 @@ def browse_facet
       params["facet_sort"] : API_OPTS["browse_sort"]
     options = {
       facet: @browse_facet.to_s,
-      facet_num: 10000,
+      facet_limit: 10000,
```

Comment:
Were there no other changes required from changing this? Or is this changed to fit with changes in the API? If so, it should be explained in the commit message. There is no context given for this change so it's kind of a mystery to me as a reviewer.